dev/CommonStyles/ContentDialog_themeresources.xaml

```diff
@@ -0,0 +1,257 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:maps="using:Windows.UI.Xaml.Controls.Maps" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+
+    <ResourceDictionary.ThemeDictionaries>
+
+        <ResourceDictionary x:Key="Default">
+
+            <StaticResource x:Key="ContentDialogBackground" ResourceKey="SystemControlPageBackgroundAltHighBrush" />
+            <StaticResource x:Key="ContentDialogBorderBrush" ResourceKey="SystemControlBackgroundBaseLowBrush" />
+            <SolidColorBrush x:Key="ContentDialogDismissLayerBackgroundBrush" Color="#1E000000" />
+
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="HighContrast">
+
+            <StaticResource x:Key="ContentDialogBackground" ResourceKey="SystemControlPageBackgroundAltHighBrush" />
+            <SolidColorBrush x:Key="ContentDialogDismissLayerBackgroundBrush" Color="#1E000000" />
+
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="Light">
+
+            <StaticResource x:Key="ContentDialogBackground" ResourceKey="SystemControlPageBackgroundAltHighBrush" />
+            <StaticResource x:Key="ContentDialogBorderBrush" ResourceKey="SystemControlBackgroundBaseLowBrush" />
+            <SolidColorBrush x:Key="ContentDialogDismissLayerBackgroundBrush" Color="#1E000000" />
+
+        </ResourceDictionary>
+
+    </ResourceDictionary.ThemeDictionaries>
+
+    <Style TargetType="ContentDialog">
+        <Setter Property="Foreground" Value="{ThemeResource ContentDialogForeground}" />
+        <Setter Property="Background" Value="{ThemeResource ContentDialogBackground}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource ContentDialogBorderBrush}" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="ContentDialog">
+                    <Border x:Name="Container" Background="{StaticResource ContentDialogDismissLayerBackgroundBrush}">
```

Comment:
The Border doesn't have background before so the smoke layer color probably doesn't come from this. 

I just took a closer look, seems like we are hard coding the smoke layer brush in code :/
https://microsoft.visualstudio.com/DefaultCollection/OS/_git/os?path=%2Fonecoreuap%2Fwindows%2Fdxaml%2Fxcp%2Fdxaml%2Flib%2FContentDialog_Partial.cpp&version=GBofficial%2F19h1_release_dcpp&line=1227&lineStyle=plain&lineEnd=1227&lineStartColumn=35&lineEndColumn=49

The code is still in OS so we can't fix it from this repo. @jevansaks should we wait for WinUI3.0 then?

Reply:
Thanks I've seen it, I also saw they have filed a bug tracking the issue.

Reply:
I'll leave the PR up until its fixed and then make the appropriate changes.